### ISSUE TITLE
sdist is missing tox.ini

### ISSUE THREAD

**Issue by mtelka** [readthedocs/readthedocs-sphinx-ext#125]
The sdist package at PyPI is missing `tox.ini`.  Please add the missing `tox.ini` to sdist to make downstream testing easier.  Thank you.

**Comment by stsewd** [comment]
Hi, this is an internal dependency of RTD, it's installed only inside our builders. Curious, where you need this package.

**Comment by mtelka** [comment]
I'm packaging `readthedocs-sphinx-ext` for OpenIndiana and I did it because it was needed as a dependency for some other package.  Not sure now which one it was because currently I'm unable to find any such dependency, but definitely it was needed in [January 2023 when I packaged it](https://github.com/OpenIndiana/oi-userland/pull/10769).

Please note that I'm not alone who thinks the `readthedocs-sphinx-ext` [deserves a package](https://repology.org/project/python:readthedocs-sphinx-ext/versions).

Thank you.

**Comment by stsewd** [comment]
I've seen other users packaging this extension because it's a dependency on tests for https://github.com/readthedocs/sphinx_rtd_theme.

https://github.com/readthedocs/sphinx_rtd_theme/blob/80de90aecff3365b1f571dedd6b5c349ec0f89cf/tox.ini#L14

**Comment by mtelka** [comment]
Yes, you are right.  Thanks for the pointer.  I failed to find this dependency because `sphinx_rtd_theme` is missing `tox.ini` too so we are not able to detect its test dependencies and list them explicitly.  We just rely on implicit `readthedocs-sphinx-ext` presence on the test host while testing `sphinx_rtd_theme`.  We should change/fix that.
